tdrs-backend/tdpservice/parsers/migrations/0007_alter_datafilesummary_datafile.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.15 on 2023-07-20 20:50
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_files', '0012_datafile_s3_versioning_id'),
+        ('parsers', '0006_auto_20230726_1448'),
```

Comment:
```suggestion
        ('parsers', '0006_auto_20230810_1500'),
```
Looks like this was recreated at a different date, updating.